lib/getRandomValue.browser.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
How does this file ever get utilized? It looks like `getRandomValue` is always loading the Node version? What am I missing?

Reply:
Never worked with browserify before - didn't realize that I needed to add an entry to the `package.json` since the tests were all passing.
Looks like when browserify sees the node crypto API being used, it goes out of its way to mock the entire API. I built a dist and was really surprised to see it blow up to `600kb`. it had stuff like:
![image](https://user-images.githubusercontent.com/11035210/44884534-b2bf5b80-ac70-11e8-92a5-a5d7d8a597af.png)
Adding an entry to the package fixes the issue.

Reply:
Ah, that explains why the tests didn't catch on fire. Good old Browserify!